catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/models.properties

```diff
@@ -18,6 +18,7 @@ circuitBreaker
 claimCheck
 combinedServiceDiscovery
 combinedServiceFilter
+concurrentConfig
```

Comment:
I assume these are left over from the old code

Reply:
Yes it is. I'm not sure why the build did not remove those. It seems to have removed others. I'll check.